### DISCUSSION TITLE
Pionia Framework Docs

### DISCUSSION THREAD

**Discussion by jet2018** [gethyas/hyas#483] [Showcase]
### Site name

Pionia Framework Official Documentation

### Public URL

https://pionia.netlify.app

### Short description

The Official documentation of Pionia PHP Framework.

### Full description

Pionia is a PHP framework that is fully RestFul. It is still under development and will soon be released fully to the rest of the world to appreciate and enjoy its cool features. 

Unlike most other traditional frameworks which use the MVC pattern, Pionia uses the Moonlight pattern to reduce development overhead so that a developer can only focus on developing amazing APIs. It fully taps into the PHP ecosystem since it supports composer. 

### Terms

- [X] I agree to the [Terms of Service](https://gethyas.com/terms/).

**Comment by h-enk** [comment]
Thanks for sharing — interesting project!

**Comment by jet2018** [comment]
You're welcome 😊 🙏
